requirements.txt

```diff
@@ -0,0 +1,8 @@
+numpy
+onnx
+onnxruntime
+onnxruntime-extensions
+psutil
+py-cpuinfo
+pydantic
+transformers
```

Comment:
Is this the package requirement for dev requirement?

Reply:
They are package requirements.  
@mengniwang95 please help to explain transformers.

Reply:
Hi @justinchuby , config.json will be loaded for LLMs at the beginning of quantization and be saved into the target folder  so that users can directly evaluate the quantized LLMs, and in this process transformers is needed https://github.com/onnx/neural-compressor/blob/9e6deacfd7a26bcbd676ee83b5f1b2aeb0cb859a/neural_compressor_ort/utils/onnx_model.py#L51

Reply:
I see. Thanks!